public/blog.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <title>Blog</title>
+  <link rel="stylesheet" href="/app.css">
+</head>
+<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+
+<body>
+  <div class="full-page">
+    <nav class="nav-bar">
+      <h1 class="nav-bar__item nav-bar__item--name">Ryan Tablada</h1>
```

Comment:
Update the headings and the home page hero to match your name and details (since this is a starting portfolio for YOU!).